apps/ftu/style/style.css

```diff
@@ -368,12 +368,14 @@ gaia-buttons.forward-only .icon-previous {
 #wifi [data-type="list"] li > p {
   pointer-events: none;
   -moz-padding-end: 6.2rem;
+  line-height: 1.33;
```

Comment:
not even a nit, just wondering about the use of the hundredth, it is very specific, and not used in any other `line-height` value along gaia. Any reason for it?